src/App.tsx

```diff
@@ -46,6 +47,11 @@ function App() {
     document
         .getElementById('favicon')!
         .setAttribute('href', `${state.spec?.SciViz.favicon_name || '/favicon.ico'}`)
+    const iframeParamMap = {
+        context: { someContext: 'hello' },
+        anotherParam: { someKey: 'someValue' },
+        stringParam: 'a string'
+    }
```

Comment:
@A-Baji Why was hard-coding this necessary?

Reply:
It is not technically hard coded. It's more of an example. Since this feature is a new/special case of interacting with a component externally through actual code rather than through the spec sheet, this is an example of how that would be done.